addons/hw_drivers/drivers/SerialBaseDriver.py

```diff
@@ -91,7 +91,7 @@ def _set_name(self):
         """Tries to build the device's name based on its type and protocol name but falls back on a default name if that doesn't work."""
 
         try:
-            name = ('%s serial %s' % (self._protocol.name, self._device_type)).title()
+            name = self._protocol.name.title()
```

Comment:
@aprieels can you check this commit?

Reply:
Yes, this change is not really useful and will change the name of connected Serial devices in stable so I don't think we need it